File: src/#1413/solution.js
var minStartValue = function(nums) {
    let cumSum = 0;
     let lowestCumSum = 0;
     
     for(let num of nums) {
         cumSum += num;
         if(cumSum < lowestCumSum) {
             lowestCumSum = cumSum;
         }
     }
     
     return lowestCumSum * -1 + 1
 };

File: src/#1413/solution.go
package solution


func minStartValue(nums []int) int{
	cumSum:=0
	lowerCumSum:=0

	for _,num:=range nums{
		cumSum+=num
		if cumSum<lowerCumSum{
			lowerCumSum=cumSum
		}
	}
	return lowerCumSum * -1 + 1
} 